lienflash/lienflash-fe
# Tm/random fixes

#### What’s this PR do?
- Adds prop types to all necessary components
- Adds redirect for bad paths
- Cleans up file structure to be more accurate (JobForm doesn't use Redux, so should be in components, not containers)

#### Where should the reviewer start?
- all the component and container files

#### How should this be manually tested?
- in terminal, run npm start. There should be no prop type errors in the browser console

#### What are the relevant tickets?
resolves #47 resolves #60
